### PR TITLE
Fix files sorting

### LINKED ISSUE #20: migrations SHOULD NOT be sorted by a file created time
As a developer, I need to have a possibility to inject a custom migration between existing. Also, it's not obvious that the last migration is run somewhere in the middle - this causes a weird bug that `migrate:rollback` rolls back not the last one but some other migration

### PR DESCRIPTION
**Pull request by vvval**
Fix the sorting order

### CODE REVIEW

File: src/Atomizer/Renderer.php [hunk 1]
```diff
@@ -119,6 +119,90 @@ public function dropTable(Source $source, AbstractTable $table): void
         );
     }
 
+    /**
```
Comment (wolfy-j) [inline src/Atomizer/Renderer.php]: Why is this in scope of this update?

**Review by wolfy-j — APPROVED** [review]

File: src/Atomizer/Renderer.php (hunk 1 above)
Comment (vvval) [reply src/Atomizer/Renderer.php]: I just run the spiral fixer. Will exclude from the PR

**Review by wolfy-j — APPROVED** [review]
